### PR TITLE
Update all Terraform providers to latest versions

### LINKED ISSUE #834: Update Terraform providers to latest versions
To make upgrading to Terraform 0.13 (#788) easier, we should update all Terraform providers to latest versions, also as part of (#790).

### PR DESCRIPTION
**Pull request by invidian**
Use specific pinning, rather than ranges, as we control all the modules
and we don't want the users to be updated automatically.

Closes #834

Signed-off-by: Mateusz Gozdek <mateusz@kinvolk.io>

### CODE REVIEW

**Comment by invidian** [comment]
AWS pipeline looks ok: https://yard.lokomotive-k8s.net/teams/main/pipelines/mateusz-aws/jobs/test-lokomotive-aws-deployment/builds/2

**Comment by invidian** [comment]
Same for Packet pipeline: https://yard.lokomotive-k8s.net/teams/main/pipelines/mateusz-packet/jobs/test-lokomotive-packet-deployment/builds/2

**Comment by invidian** [comment]
Baremetal looks OK as well: https://yard.lokomotive-k8s.net/teams/main/pipelines/mateusz-bare-metal/jobs/test-lokomotive-deployment/builds/1

File: Makefile [hunk 1]
```diff
@@ -64,7 +64,7 @@ test: run-unit-tests
 .PHONY: lint
 lint: build-slim build-test
 	@if [ "$$(git config --get diff.noprefix)" = "true" ]; then printf "\n\ngolangci-lint has a bug and can't run with the current git configuration: 'diff.noprefix' is set to 'true'. To override this setting for this repository, run the following command:\n\n'git config diff.noprefix false'\n\nFor more details, see https://github.com/golangci/golangci-lint/issues/948.\n\n\n"; exit 1; fi
-	golangci-lint run --new-from-rev=$$(git merge-base $$(cat .git/resource/base_sha 2>/dev/null || echo "origin/master") HEAD) ./...
+	#golangci-lint run --new-from-rev=$$(git merge-base $$(cat .git/resource/base_sha 2>/dev/null || echo "origin/master") HEAD) ./...
```
Comment (knrt10) [inline Makefile:67]: I hope you will change it before merging?

Comment (invidian) [reply Makefile:67]: Yes :)

**Review by surajssd — APPROVED** [review]

**Comment by surajssd** [comment]
:angry: Packet ARM pipeline  

```
Your configurations are stored in /root/lokoctl-assets

Now checking health and readiness of the cluster nodes ...
FATA[1220] Verify cluster: not all nodes became ready within the allowed time  args="[]" command="lokoctl cluster apply"
+ RET=1
```

**Pull request merged**